src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -19,6 +20,8 @@
  *      specifically for rank questions.
  */
 public class FeedbackRankQuestionUiTest extends FeedbackQuestionUiTest {
```

Comment:
We now lost the `TODO` where we supposed to do backend validation.

Can you add something like `//TODO: Backend validation. Blocked by #8646`